### PR TITLE
[srvls][RHDEVDOCS-2749] Updating sink, channels, sources documentation

### PR DESCRIPTION
**Pull request by abrennan89**
Preview link: https://deploy-preview-32453--osdocs.netlify.app/openshift-enterprise/latest/serverless/channels/serverless-channels.html

Applies for 4.6+

### CODE REVIEW

**Comment by abrennan89** [comment]
To do:

Add details about sinks to 

1. https://deploy-preview-32453--osdocs.netlify.app/openshift-enterprise/latest/serverless/event_sources/serverless-apiserversource.html#apiserversource-kn_context
2. https://deploy-preview-32453--osdocs.netlify.app/openshift-enterprise/latest/serverless/event_sources/serverless-pingsource.html#serverless-pingsource-kn_serverless-pingsource
3. https://deploy-preview-32453--osdocs.netlify.app/openshift-enterprise/latest/serverless/event_sources/serverless-sinkbinding.html#serverless-sinkbinding-kn_serverless-sinkbinding
4. https://deploy-preview-32453--osdocs.netlify.app/openshift-enterprise/latest/serverless/event_sources/serverless-kafka-source.html#serverless-kafka-source-kn_serverless-kafka-source
5. https://deploy-preview-32453--osdocs.netlify.app/openshift-enterprise/latest/serverless/channels/connect-channels-event-source.html

Add info about custom / other sink types to https://deploy-preview-32453--osdocs.netlify.app/openshift-enterprise/latest/serverless/event_sources/srvls-connecting-source-consumers.html#specifying-sink-flag-kn_srvls-connecting-source-consumers / https://deploy-preview-32453--osdocs.netlify.app/openshift-enterprise/latest/serverless/cli_reference/kn-flags-reference.html

File: _topic_map.yml [hunk 1]
```diff
@@ -2930,6 +2927,17 @@ Topics:
     File: serverless-sinkbinding
   - Name: Using a Kafka source
     File: serverless-kafka-source
+  - Name: Connecting event sources to consumers
+    File: srvls-connecting-source-consumers
+# Event sources
```
Comment (abrennan89) [inline _topic_map.yml:2932]: remove event sources

File: _topic_map.yml [hunk 2]
```diff
@@ -2930,6 +2927,17 @@ Topics:
...
+- Name: Channels
+  Dir: channels
+  Topics:
+# Channels
```
Comment (abrennan89) [inline _topic_map.yml:2936]: move to event sources line

File: _topic_map.yml [hunk 3]
```diff
@@ -2930,6 +2927,17 @@ Topics:
     File: serverless-sinkbinding
   - Name: Using a Kafka source
     File: serverless-kafka-source
+  - Name: Connecting event sources to consumers
+    File: srvls-connecting-source-consumers
```
Comment (abrennan89) [inline _topic_map.yml:2931]: not sure about name, consumer, sink?

File: modules/specifying-sink-flag-kn.adoc [hunk 4]
```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_sources/srvls-connecting-source-consumers
+// * serverless/cli_reference/
+
+[id="specifying-sink-flag-kn_{context}"]
+= Specifying a sink for a resource by using a Knative CLI (`kn`) command
```
Comment (abrennan89) [inline modules/specifying-sink-flag-kn.adoc:7]: shorter / better title?

**Comment by mgencur** [comment]
Looks good to me.
QE Approved.

File: modules/specifying-sink-flag-kn.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+// * serverless/cli_reference/
+// * serverless/event_sources/serverless-pingsource
+// * serverless/event_sources/serverless-apiserversource
+// * serverless/event_sources/serverless-kafka-source
+// * serverless/event_sources/serverless-sinkbinding
+
+[id="specifying-sink-flag-kn_{context}"]
+= Knative CLI (kn) --sink flag
+
+When you create an event-producing custom resource by using the Knative (`kn`) CLI, you can specify a sink where events will be sent to from that resource, by using the `--sink` flag.
```
Comment (bmcelvee) [inline modules/specifying-sink-flag-kn.adoc:13]: s/will be/are

File: serverless/event_sources/srvls-connecting-source-sinks.adoc [hunk 6]
```diff
@@ -0,0 +1,20 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="srvls-connecting-source-sinks"]
+= Connecting event sources to sinks
+include::modules/common-attributes.adoc[]
+:context: srvls-connecting-source-sinks
+
+toc::[]
+
+When you xref:../../serverless/event_sources/knative-event-sources.adoc#knative-event-sources[create an event source], you can specify a sink where events will be sent to from the source. A sink is an addressable resource that can receive incoming events from other Knative Eventing resources. Knative services, channels, and brokers are all examples of sinks.
```
Comment (bmcelvee) [inline serverless/event_sources/srvls-connecting-source-sinks.adoc:9]: s/will be/are

**Comment by bmcelvee** [comment]
Couple of suggestions, otherwise LGTM

**Pull request merged**